.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
+  run_smoke_test:
+    working_directory: ~/teacher-vacancy-service
+    docker:
+      - image: circleci/ruby:2.6.1-node-browsers-legacy
+        environment:
+        RAILS_ENV: test
+    steps:
+      - checkout
+      - run: gem install bundler
+      - run: bundle install
+      - run: bundle exec rspec --tag smoke_test spec/smoke_test/
```

Comment:
Is there a need for the directory path `spec/smoke_test/` or is having the `--tag smoke_test` argument sufficient? Rspec should scan all possible tests and would only use any with `smoke_test: true` tagged anyway. I guess it may speed up file traversing when searching for files to run.

Reply:
In this case, it directly looks into the spec/smoke_test folder instead of searching through all spec files in the project, the later also causing errors since most of the test requires the test server to be up